src/test/java/com/snack/news/service/TopicServiceTest.java

```diff
@@ -19,10 +19,13 @@
 import java.util.List;
 import java.util.Optional;
 
+import static com.snack.news.matcher.ContainsInAnyOrder.containsInAnyOrder;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.*;
+import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4677893)